contrib/nautobot.service

```diff
@@ -14,6 +14,8 @@ PIDFile=/var/tmp/nautobot.pid
 WorkingDirectory=/opt/nautobot
 
 ExecStart=/opt/nautobot/bin/nautobot-server start --pidfile /var/tmp/nautobot.pid --ini /opt/nautobot/uwsgi.ini
+ExecStop=/opt/nautobot/bin/nautobot-server start --stop /var/tmp/nautobot.pid
+ExecReload=/opt/nautobot/bin/nautobot-server start --reload /var/tmp/nautobot.pid
```

Comment:
Hah, that's a bit inelegant to call `nautobot-server start --stop`. I don't have any bright ideas on what we could change `start` to though, so just consider this an observation.

Reply:
I had the same observation, and @nniehoff suggested that we perhaps just call this `uwsgi` vs. `start`.

Reply:
I'd be fine with that - though given how much more that'll impact other documentation, maybe make that change as a separate PR?